app/src/utils/utils.ts

```diff
@@ -426,3 +426,20 @@ export function parseGsoState(buf: Buffer) {
     lockupPeriodEnd,
   };
 }
+
+export async function getJupPriceAPI(baseMint: PublicKey): Promise<number> {
```

Comment:
is this not used?

Reply:
Not used, left it since may want to switch over once it starts returning the correct price. It's been stuck at 0.032 for ahwile